core/assets/bundles/bundle_fr.properties

```
@@ -635,16 +661,36 @@ category.multiplayer.name = Multijoueur
 command.attack = Attaque
 command.rally = Rassembler
 command.retreat = Retraite
+placement.blockselectkeys = \n[lightgray]Touche: [{0},
 keybind.clear_building.name = Effacer les constructions
 keybind.press = Appuyer sur une touche...
 keybind.press.axis = Appuyer sur un axe ou une touche...
 keybind.screenshot.name = Capture d'écran
-keybind.move_x.name = Mouvement x
-keybind.move_y.name = Mouvement y
+keybind.toggle_power_lines.name = Montrer/Cacher les connections d'énergie
```

First letters of each word should be in uppercase and change `Montrer` to `Afficher` like [here](https://github.com/Anuken/Mindustry/blob/7f2a529c081cabd191e53e9358ecf029648a3194/core/assets/bundles/bundle_fr.properties#L667).
```suggestion
keybind.toggle_power_lines.name = Montrer/Cacher les Connections d'Énergie
```
*See also: https://github.com/Anuken/Mindustry/blob/7f2a529c081cabd191e53e9358ecf029648a3194/core/assets/bundles/bundle.properties#L669*

---

And maybe it's `Power Lasers`:
```suggestion
keybind.toggle_power_lines.name = Montrer/Cacher les Lasers d'Énergie
```